addon/components/paper-button.js

```diff
@@ -7,14 +7,22 @@ import ColorMixin from 'ember-paper/mixins/color-mixin';
 const { computed } = Ember;
 
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
-  attributeBindings: ['type'],
   type: 'button',
-  tagName: 'button',
+  tagName: computed('href', function() {
```

Comment:
Also, I think we should encapsulate the logic of testing href in a `isLinkButton` computed. That way, if we ever want to change the test we do it in just onde place.